test/audio_player_test.c

```diff
@@ -93,11 +161,18 @@ TEST_CASE("audio player states and callbacks are correct", "[audio player]")
 {
     audio_player_callback_event_t event;
 
-    esp_err_t ret = i2s_init(I2S_NUM_0, 44100);
+    /* Configure I2S peripheral and Power Amplifier */
+    i2s_std_config_t std_cfg = {
+        .clk_cfg = I2S_STD_CLK_DEFAULT_CONFIG(44100),
```

Comment:
is this and the below function/macro coming from that i2s_std.h include? Is that something new in esp-idf 5 or local to esp-box? I don't see it here in my v4 esp-idf.